app-config.prod.yaml

```
@@ -21,6 +21,20 @@ backend:
   cors:
     origin: https://service-catalog.operate-first.cloud
 
+techdocs:
+  builder: 'local'
+  generator:
+    runIn: 'local'
+  publisher:
+    type: 'awsS3'
+    awsS3:
+      bucketName: ${BUCKET_NAME}
+      region: ${BUKCET_REGION}
+      endpoint: ${BUCKET_ENDPOINT}:${BUCKET_PORT}
```

Protocol is missing here, see:
https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#constructor-property
```suggestion
      endpoint: https://${BUCKET_ENDPOINT}:${BUCKET_PORT}
```